src/components/SliderBox/components/SliderArrow/SliderArrow.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import './SliderArrow.scss';
+
+const enum ArrowDirection {
+  Previous = 'previous',
+  Next = 'next',
+}
+
+interface SliderArrowProps {
+  direction: string;
+}
+
+export const SliderArrow = ({ direction }: SliderArrowProps) => (
+  <>
+    {ArrowDirection.Previous === direction ? (
+      <ArrowBackIosIcon
+        className={`slider-card-icon-${direction}-arrow`}
+        color="primary"
+      />
+    ) : ArrowDirection.Next === direction ? (
+      <ArrowForwardIosIcon
+        viewBox="0 0 14 24"
+        className={`slider-card-icon-${direction}-arrow`}
+        color="primary"
+      />
+    ) : (
+      ''
```

Comment:
1. why do we need this part? 
2. for what purposes `viewBox` props is used?

Reply:
Only this was can align svg icons into center for their looks good without using translate for only 1 icon into styles